ColumnSet.js

```diff
@@ -182,6 +182,19 @@ define([
 			this.on('.dgrid-column-set:dgrid-cellfocusin', function (event) {
 				self._onColumnSetCellFocus(event, this);
 			});
+
+			if ('expand' in this) {
+				aspect.after(this, 'expand', this.afterRowExpand, true);
```

Comment:
I would probably vote for just including the function internally/anonymously here rather than separating it.  It shouldn't be public, and even if we underscore-prefixed it, we'd want to pick a name that is unlikely to collide with something else.  In this case, I don't think it's worth exposing.